code/datums/quirks/negative.dm

```diff
@@ -6,6 +6,188 @@
+	medical_record_text = "This patient is observed to take 25% more physical and mental damage when applied at once"
+	hardcore_value = 4
+
+/datum/quirk/major_black/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.physiology.black_mod += 0.25
+
+/datum/quirk/fatal_black
+	name = "Fatal black damage weakness"
+	desc = "You take twice as much BLACK damage"
+	value = -4
+	gain_text = "<span class='danger'>You feel extremelly weak to BLACK damage</span>"
+	medical_record_text = "This patient is observed to take double the physical and mental damage when applied at once"
+	hardcore_value = 8
+
+/datum/quirk/fatal_black/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.physiology.black_mod += 1.00
+
+	// Pale damage quirks start
+
+/datum/quirk/minor_pale
+	name = "Minor pale damage weakness"
+	desc = "You take 10% more PALE damage"
+	value = -1
+	gain_text = "<span class='danger'>You feel a bit weaker to PALE damage</span>"
+	medical_record_text = "This patient is observed to take 10% more physical damage to certain weaponry"
+	hardcore_value = 2
+
+/datum/quirk/minor_pale/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.physiology.pale_mod += 0.10
+
+/datum/quirk/major_pale
+	name = "Major pale damage weakness"
+	desc = "You take 25% more PALE damage"
+	value = -2
+	gain_text = "<span class='danger'>You feel weaker to PALE damage</span>"
+	medical_record_text = "This patient is observed to take 25% more physical damage to certain weaponry"
+	hardcore_value = 4
+
+/datum/quirk/major_pale/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.physiology.pale_mod += 0.25
+
+/datum/quirk/fatal_pale
+	name = "Fatal pale damage weakness"
+	desc = "You take twice as much PALE damage"
+	value = -4
+	gain_text = "<span class='danger'>You feel extremelly weak to PALE damage</span>"
+	medical_record_text = "This patient is observed to take double the physical damage to certain weaponry"
+	hardcore_value = 8
+
+/datum/quirk/fatal_pale/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.physiology.pale_mod += 1.00
+
+	// Challange quirks start (they cause players to restort to a unique playstyle.)
+/datum/quirk/guns
+	name = "Challenge: No guns"
+	desc = "You refuse to use guns, afterall the bullets cost more than your entire life AND some ego fire like 400 rounds per minute. Surelly all other agents are in debt but you know better."
+	value = -1
+	mob_trait = TRAIT_NOGUNS
+	gain_text = "<span class='danger'>You can't use ranged EGO nor basic guns.</span>"
+	medical_record_text = "This patient refuses to use guns, understandable considering taxes."
+	hardcore_value = 0
+
+/datum/quirk/healing
+	name = "Challenge: No medipens"
+	desc = "Chemicals inside you will never be processed, meaning you cant heal from medipens and such (but also dont have to eat). better hope clerks know how to use those R.A.C.K's if you fall into crit."
+	value = -1
+	mob_trait = TRAIT_NOMETABOLISM
```

Comment:
This might cause issues later on, can i suggest we make a new trait and if the person has that trait the healing chems dont work? I think we can actually put a If in our usual chemicals for that.

Reply:
could definetelly try that, i'll look into it later on

Reply:
this took way longer than i can admit